src/ops/info.rs

```diff
@@ -1,17 +1,20 @@
 //! The info callable is for printing
 
+use crate::builder::OutputPaths;
 use crate::ops::error::{ok, OpResult};
-use crate::VERSION_BUILD_REV;
-use std::path::PathBuf;
+use crate::project::{roots::Roots, Project};
+use slog_scope::info;
 
 /// See the documentation for lorri::cli::Command::Info for more
 /// details.
-pub fn main(shell_nix: PathBuf) -> OpResult {
-    println!("lorri version: {}", VERSION_BUILD_REV);
-    println!("Lorri Project Configuration");
-    println!();
-
-    println!("expression: {}", shell_nix.display());
-
+pub fn main(project: Project) -> OpResult {
+    info!("lorri version: {}", crate::VERSION_BUILD_REV);
+    let root_paths = Roots::from_project(&project).paths();
+    let OutputPaths { shell_gc_root } = &root_paths;
+    if root_paths.all_exist() {
+        info!("gc roots exist"; "shell_gc_root" => ?shell_gc_root.as_os_str());
+    } else {
+        info!("gc roots do not exist"; "shell_gc_root" => ?shell_gc_root.as_os_str());
+    }
```

Comment:
I think since the entire purpose of the info command is to print this info, it should use `println` instead of a logging mechanism.

Reply:
Fixed.